dkc/settings.py

```diff
@@ -41,6 +41,8 @@ class DkcMixin(ConfigMixin):
     ACCOUNT_AUTHENTICATION_METHOD = 'username_email'
 
     DKC_DEFAULT_QUOTA = 3 << 30  # 3 GB
+    DKC_AUTHORIZED_UPLOAD_EXPIRATION_DAYS = 7
+    DKC_SPA_URL = 'https://dkc-next-web.girderops.net/'
```

Comment:
```suggestion
    DKC_SPA_URL = values.Value(environ_required=True)
```

This will require us to set `DJANGO_DKC_SPA_URL` at deployment time, allowing it to be determined by the deployment code.

---

Within `DevelopmentConfiguration`, I'd add:
```
DKC_SPA_URL = values.Value('http://localhost:8080/`)
```
which will give the value a sensible default, but still allow it to be overridden via environment variable.

---

Within `TestingConfiguration`, you can set:
```
DKC_SPA_URL = 'http://dkc-web.test/'
```
which will ensure it's set if it ever gets executed in a test, and gives us a deterministic value if we ever want a test to make assertions about it.